src/pybind/ceph_volume_client.py

```diff
@@ -318,7 +317,7 @@ def recover(self):
 
         log.debug("Recovered from partial auth updates (if any).")
 
-    def _recover_auth_meta(auth_id, auth_meta):
+    def _recover_auth_meta(self, auth_id, auth_meta):
```

Comment:
@onyb I've created a bug tracker for this issue that I'd like work to on. Please let me know if you're OK with me taking this up and other broken recovery related problems?
http://tracker.ceph.com/issues/17216